src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -0,0 +1,30 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+
+import * as config from "../../config";
+import {
+  BeneficialOwnerTypeChoice,
+  BeneficialOwnerTypeKey,
+} from "../../model/beneficial.owner.type.model";
+import { getBeneficialOwnerType } from "../../utils/beneficial.owner.type";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  getBeneficialOwnerType(req, res, next, false);
+};
+
+export const post = (req: Request, res: Response) => {
+  logger.debugRequest(req, `POST ${req.method} ${req.route.path}`);
+
+  return res.redirect(getNextPage(req.body[BeneficialOwnerTypeKey]));
+};
+
+// With validation in place we will only have 3 choices
+const getNextPage = (beneficialOwnerTypeChoices?: BeneficialOwnerTypeChoice): string => {
+  if (beneficialOwnerTypeChoices === BeneficialOwnerTypeChoice.individual) {
```

Comment:
could use switch statement

Reply:
have not seen switch statements used anywhere, not sure if there is any specific reason, so stuck with if , added an else so it does not seem like default but with validation it should always work

Reply:
if is fine but it still looks like it defaults to GOV_URL. What if someone adds another type and forgets to change if? Should it be "else if (beneficialOwnerTypeChoices === BeneficialOwnerTypeChoice.governmentOrPublicOwner) {"?

Reply:
or ok to default to individual type?